modules/sbo-advanced-binding-options.adoc

```diff
@@ -115,4 +121,66 @@ spec:
 <1> Name of the backing service resource.
 <2> Optional identifier.
 <3> Generated JSON name that is to be projected as the file content or environment value. The JSON name contains the attributes of the backing service custom resource.
-<4> Generated JSON value that is to be projected as the file content or environment value. The JSON value contains the attributes of the backing service custom resource.
\ No newline at end of file
+<4> Generated JSON value that is to be projected as the file content or environment value. The JSON value contains the attributes of the backing service custom resource.
+
+[id="binding-workloads-using-a-label-selector_{context}"]
+== Binding workloads using a label selector
+You can use a label selector to specify the workload that is being bound. If you declare a service binding using the label selectors to pick up workloads, the {servicebinding-title} periodically attempts to find and bind new workloads that match the given label selector.
+
+For example, you may want to bind a service to every `Deployment` in a namespace with the `environment: production` label set in the `application` field. By setting an appropriate label selector, you can enable the {servicebinding-title} to bind each of these workloads with one `ServiceBinding` resource.
+
+.Example of `ServiceBinding` CR in the `binding.operators.coreos.com/v1alpha1` API:
+[source,yaml]
+----
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: multi-application-binding
+  namespace: service-binding-demo
+spec:
+  application:
+    labelSelector: <1>
+      matchLabels:
+        environment: production
+    group: apps
+    version: v1
+    resource: deployments
+  services:
+    group: ""
+    version: v1
+    kind: Secret
+    name: super-secret-data
+----
+<1> Specifies the workload that is being bound.
+
+.Example of `ServiceBinding` CR in the `servicebinding.io` API:
+[source,yaml]
+----
+apiVersion: servicebindings.io/v1alpha3
+kind: ServiceBinding
+metadata:
+  name: multi-application-binding
+  namespace: service-binding-demo
+spec:
+  workload:
+    selector: <1>
+      matchLabels:
+        environment: production
+    apiVersion: app/v1
+    kind: Deployment
+  service:
+    apiVersion: v1
+    kind: Secret
+    name: super-secret-data
+----
+<1> Specifies the workload that is being bound.
+
+[IMPORTANT]
+====
+Currently, it is forbidden to attempt a binding with the following fields defined:
+
+* Both `name` and `labelSelector` in the `binding.operators.coreos.com/v1alpha1` API
+* Both `name` and `selector` in the `servicebinding.io` API (Spec API)
+
+Any attempt to do so will result in an error.
```

Comment:
```suggestion
* The `name` and `labelSelector` fields in the `binding.operators.coreos.com/v1alpha1` API
* The `name` and `selector` fields in the `servicebinding.io` API (Spec API)
```

Reply:
If you keep line 185, please rephrase it to avoid using future tense.

Reply:
Fixed